pywavelets/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: pywavelets
+  version: "0.3.0"
+
+source:
+  git_url: https://github.com/rgommers/pywt.git
+  git_rev: v0.3.0
+
+requirements:
+  build:
+    - python 2.6*|2.7*|>=3.3
+    - cython
+    - numpy >=1.6.2
+
+  run:
+    - python
+    - numpy
+
+test:
+  requires:
+    - nose
+
+about:
+  home: http://www.pybytes.com/pywavelets/
```

Comment:
We've migrated away from ``pybytes.com``. I suggest using https://github.com/PyWavelets/pywt here. Or otherwise http://pywavelets.readthedocs.org is also OK.